src/command_modules/azure-cli-extension/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.0.10
+++++++
+A bug fix in the --extra-index-url argument in az extension add and update commands.
+
```

Comment:
We haven't released 0.0.9 yet so you can just modify the entry you created prev. for that one.